src/app/Modules/dashboard/dashboard.component.md

```diff
@@ -0,0 +1,56 @@
+# Dashboard Component
+
+Dashboard is the default landing page for the users after they are able to successfully login.
+
+![Default Dashoard](images/dashboard.png)
+
+- `Dashboard` contains lot of sections where users can find minimal required information.
+- The top section of the dashboard contains the list of sites with images and some basic information of that particular site.
+- Users can switch between sites by clicking on the respective cards and in the bottom the site related information is displayed.
+- Immediately after the sites, the bandwidth information of the site is displayed.
+- `Highlights` of that particular site are also visible next to the bandwidth information.
+- Below the bandwidth the `slices` in the selected site are displayed in the list view.
+- Each `Slice` can be expanded using the plus icon to display the detailed view of the slice.
+
+  ![Slice expand view](images/dashboard-expand-view.png)
+
+- In the expanded view of the `Slice` we can see the device groups and the devices in the respective device group can be viewed.
+- In the middle section of the `Slice` we can see the **Latency, Bandwidth and Packet Loss** of the slice.
+- Users can also edit the slices. The edit view of the Slice looks as shown below:
+
+  ![Slice edit view](images/dashboard-edit-view.png)
+
+- In the edit view of the `Slice` users can see the option to add and delete device groups from the `Slice`.
+- In addition users can also handle `Services` of the slice.
+- In the `Physical view` of the site view users can see the `Small Cell` view as shown below:
+
+  ![Small Cell View](images/dashboard-physical-view.png)
+
+- The operations on the Small Cell view is similar to the [Small Cells](/src/app/Modules/settings/small-cell/small-cell.component.md) view.
```

Comment:
I think it will need a relative path to make sure it works on Github too 
e.g. `../settings/small-cell/small-cell.component.md`